### ISSUE TITLE
Unclear ERROR: Send using sockets failed with system error: write: Broken pipe  

### ISSUE THREAD

**Issue by vojenfr** [precice/precice#1376]
Fedora 33 (Container Image) ==> https://github.com/TEFEdotCC/Containered-OpenFOAM.com/blob/v2012/Dockerfile
with precice ==> version 2.2.1

The test "/opt/preCICE/openfoam-adapter/tutorials/CHT/water-heater/icoReactingMultiphaseInterFoam-laplacianFoam/Allrun" 
is ok with OpenMPI on one node , but not ok with **two** nodes (> 16 CPU) :

ExecutionTime = 326.28 s  ClockTime = 334 s                                                                                                   

---[preciceAdapter] [DEBUG] Writing coupling data...                                                                                          
---[preciceAdapter] [DEBUG] Advancing preCICE...                                                                                              
---[precice]  relative convergence measure: relative two-norm diff = 0, limit = 1e-06, normalization = 6328.31, conv = true                   
---[precice]  relative convergence measure: relative two-norm diff = 0, limit = 1e-06, normalization = 6328.31, conv = true                   
---[precice]  relative convergence measure: relative two-norm diff = 0, limit = 1e-06, normalization = 6328.31, conv = true                   
---[precice]  relative convergence measure: relative two-norm diff = 0, limit = 1e-06, normalization = 6328.31, conv = true                   
---[precice]  All converged                                                                                                                   
---[precice] ERROR:  Send using sockets failed with system error: write: Broken pipe 

just 4 lines was changed on precice-config.xml:
![Sans titre](https://user-images.githubusercontent.com/6704053/179743666-f70ea09f-07f8-4ddc-a4cf-44e44e9f9bbd.png)

and not start with the Communication : <m2n:mpi ...

**Comment by uekerman** [comment]
> ERROR: Send using sockets failed with system error: write: Broken pipe

means typically that the "other side" crashed. What behavior do you get there?

**Comment by vojenfr** [comment]
No information in syslog/messages on the second node (other side) , but same test seems work with "**libprecice.so.2.4.0**"
How get more information on the second node ?

**Comment by MakisH** [comment]
> How get more information on the second node ?

In what @uekerman wrote, "other side" means the other participants of the simulation. I understand that you define `Fluid`, `Heater1`, ..., `Heater4`. One of these participants probably had reported some error.

This does not look like a preCICE bug to me, but the error message is indeed not helpful. We did already improve it in v2.2.0, but we can give more hints. I extended the error message with a hint in #1383, since you are not the first person that gets confused by this.

Note that you are running a third-party setup that we don't know much about. If the problem still persists, I think it would fit more as a support request on the forum: https://precice.discourse.group/ :grinning:
